src/main/java/org/openmaptiles/layers/Transportation.java

```diff
@@ -415,6 +517,15 @@ int getMinzoom(Tables.OsmHighwayLinestring element, String highwayClass) {
         case FieldValues.CLASS_SERVICE -> isDrivewayOrParkingAisle(service(element.service())) ? 14 : 13;
         case FieldValues.CLASS_TRACK, FieldValues.CLASS_PATH -> routeRank == 1 ? 12 :
           (z13Paths || !nullOrEmpty(element.name()) || routeRank <= 2 || !nullOrEmpty(element.sacScale())) ? 13 : 14;
+        case FieldValues.CLASS_TRUNK -> {
+          // trunks in some networks to have same min. zoom as highway = "motorway"
+          String clazz = routeRelations.stream()
+            .map(RouteRelation::networkType)
+            .filter(Objects::nonNull)
+            .map(Enum::toString)
+            .anyMatch(TRUNK_AS_MOTORWAY_BY_NETWORK::contains) ? FieldValues.CLASS_MOTORWAY : FieldValues.CLASS_TRUNK;
```

Comment:
could `TRUNK_AS_MOTORWAY_BY_NETWORK` store the enum values and compare against them instead of the strings?